src/main/kotlin/org/rust/ide/annotator/RsHighlightingMutableAnnotator.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator
+
+import com.intellij.lang.annotation.AnnotationHolder
+import com.intellij.lang.annotation.Annotator
+import com.intellij.lang.annotation.HighlightSeverity
+import com.intellij.psi.PsiElement
+import org.rust.ide.colors.RsColor
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.RsElement
+import org.rust.lang.core.psi.ext.RsNameIdentifierOwner
+import org.rust.lang.core.psi.ext.ancestorStrict
+import org.rust.lang.core.psi.ext.mutability
+import org.rust.lang.core.types.ty.TyReference
+import org.rust.lang.core.types.type
+import org.rust.openapiext.isUnitTestMode
+
+class RsHighlightingMutableAnnotator : Annotator {
+    private val HIGH_PRIORITY_HIGHLIGHTING = HighlightSeverity(
+        "HIGH_PRIORITY_HIGHLIGHTING",
```

Comment:
Let's rename it to show that it's about mutable variables.
And move it into `companion object`